Source/control.cpp

```diff
@@ -114,29 +114,35 @@ bool IsRightPanelOpen()
 	return invflag || sbookflag;
 }
 
+constexpr int SdlRectYAdjustment = 16;
+
 constexpr Size IncrementAttributeButtonSize { 41, 22 };
 /** Maps from attribute_id to the rectangle on screen used for attribute increment buttons. */
-Rectangle ChrBtnsRect[4] = {
+Rectangle CharButtonRect[4] = {

```

Comment:
Since it's an array the name should probably be plural, also I think we can drop the type from the name.
```suggestion
Rectangle CharButtons[4] = {
```